### ISSUE TITLE
Support for password-protected archives?

### ISSUE THREAD

**Issue by hkjolhede** [r-lib/archive#8]
Do you have any plans for supporting password-protected archives, such as zip or 7zip files?

**Comment by jimhester** [comment]
No plans, but libarchive does support encrypted files (https://github.com/libarchive/libarchive/issues/579) so it is possible to add.

**Comment by hkjolhede** [comment]
Yes, that was what I figured. I'll see if i may find time to maybe do something and create a PR.
